text.js

```diff
@@ -254,15 +254,26 @@ FastTextDecoder.prototype['decode'] = function(buffer, options={stream: false})
     throw new Error(`Failed to decode: the 'stream' option is unsupported.`);
   }
 
-  // Accept Uint8Array instances as-is.
-  let bytes = buffer;
+  let bytes;
```

Comment:
I'd rather default to the passed buffer, in case folks want to decode an array-like.